docs/index.html

```diff
@@ -480,6 +480,37 @@ <h2>Threat Model</h2>
       </section>
     </section>
 
+    
+    
+    <section class="informative">
+      <h2>Disaster Recovery</h2>
+      <p>
+        Industry best practices related to disaster recovery should be followed
+        by implementations compliant with this specification, specifically at a minimum
+        guidance from the following whitepapers and specifications SHOULD be used:
+        <ul>
+          <li>
+            <a href="https://csrc.nist.gov/publications/detail/white-paper/2020/04/24/protecting-data-from-ransomware-and-other-data-loss-events/final">
+              Protecting Data from Ransomware and Other Data Loss Events
+            </a>
+          </li>
+          <li>
+            <a href="https://nvlpubs.nist.gov/nistpubs/cswp/NIST.CSWP.10012020-draft.pdf">
+              Securing Data Integrity Against Ransomeware Attacks
+            </a>
+          </li>
+          <li>
+            <a href="https://csrc.nist.gov/publications/detail/sp/1800-11/final">
+              NIST SP 1800-11 - Data Integrity: Recovering from Ransomware and Other Destructive Events
+            </a>
+          </li>
+        </ul>
```

Comment:
Probably need another publication on recovery from outages or something that isn't explicitly an external attack, something maybe like this where its more general principles? https://docs.aws.amazon.com/whitepapers/latest/disaster-recovery-workloads-on-aws/disaster-recovery-options-in-the-cloud.html , although this is AWS only so a different source might be better.

Reply:
https://github.com/w3c-ccg/traceability-interop/issues/52